### PR TITLE
Add information about GPU-focused AMI

### PR DESCRIPTION
**Pull request by lukedeo**
A lot of GPU related software can have a steep learning curve w.r.t. installation and configuration. Amazon has a pre-baked AMI to get started at no up front cost, at the risk of a bit of lock-in if you use these AMIs in production.

### CODE REVIEW

**Comment by jlevy** [comment]
Great addition. Two possible small improvements:
- Should this bullet could be moved up, so it's up one level (a 2nd level bullet before "As with any expensive...")?
- This lock-in is because of Amazon Linux as well as the software configs, right? Might be worth mentioning that there is lock-in from that, and putting ⛓ at the front of the bullet.

Over to @max-zanko (ec2 topic editor) for 2nd pair of eyes.

**Comment by jlevy** [comment]
Cool! LGTM. @max-zanko ?

File: README.md [hunk 1]
```diff
@@ -756,11 +756,13 @@ EC2
...
+
+	-	There are [three generations](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using_cluster_computing.html) of GPU-enabled instances available:
 		- Third generation P2 series offers NVIDIA K80 GPUs in 1, 8 and 16 GPU configurations targeting machine learning and scientific workloads.
 		- Second generation G2 series offers NVIDIA K520 GPUs in 1 or 4 GPU configurations targeting graphics and video encoding.
 		- First generation CG1 instances are still available in some regions in a single configuration with a NVIDIA M2050 GPU.
-  - 🔹As with any expensive EC2 instance types, [Spot instances can offer significant savings](#ec2-cost-management) with GPU workloads when interruptions are tolerable.
+	-	⛓ AWS offers an [AMI](https://aws.amazon.com/marketplace/pp/B01M0AXXQB?qid=1475211685369&sr=0-1&ref_=srh_res_product_title) (based on Amazon Linux) with most NVIDIA drivers and ancillary software (CUDA, CUBLAS, CuDNN, TensorFlow) installed to lower the barrier to usage. Note, however, that this leads to lock-in due to Amazon Linux and the fact that you have no direct access to software configuration or versioning.
```
Comment (max-zanko) [inline README.md]: Is there an open version of a similar setup (instance creation script or an AMI) that can be used instead?

**Comment by jlevy** [comment]
Any other changes on this @max-zanko ? It's ready to go as far as I'm concerned.

**Pull request merged**